### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by eltociear**
intructions -> instructions

### CODE REVIEW

**Review by gz83 — APPROVED** [review]
LGTM

Please wait for Alex to give his LGTM and let him merge it

@eltociear @Alex313031

**Review by Alex313031 — APPROVED** [review]
@eltociear Thanks. Merged.

**Pull request merged**
